### ISSUE TITLE
When looping keep slide animation same direction

### ISSUE THREAD

**Issue by pixelspencil** [alvarotrigo/pagePiling.js#112]
Hi - great plugin.

Just wondering if there is a way when the plugin is set to loop top/bottom that the animation remains the same, currently when you hit the last slide and then scroll down to next one which is slide 1 it animates from top to bottom while naturally to me I'd like it to keep animating bottom to top when scrolling in that direction and not changing when looping - hope that makes sense.

If you take this example: http://alvarotrigo.com/pagePiling/examples/looping.html#page3

If you scroll down - when looping the slides - I'd like the animation to be bottom > top
if you scroll up - when looping the slides - I'd like the animation to be top > bottom

Is this possible?

Thanks in advance!

**Comment by alvarotrigo** [comment]
The library doesn't provide such option at the moment I'm afraid.

**Comment by pixelspencil** [comment]
Ah that's a shame - is there no way to code it in, e.g. like when first/last slide is scrolled past keep direction the same? No way it could be custom coded by me using some of the params and custom js?

**Comment by alvarotrigo** [comment]
> No way it could be custom coded by me using some of the params and custom js?

Of course, the library is open source. Fork it and modify it if you wish.

**Comment by pixelspencil** [comment]
Thank you - I will mess about with it when I have a change - great work btw.

**Comment by desarrollo2color2** [comment]
Hi there, did you find any solution about this? this is a great plugin and this would be awesome to have.

**Comment by alvarotrigo** [comment]
@desarrollo2color2 I would encourage you to make use of my other library, [fullpage.js](https://github.com/alvarotrigo/fullPage.js).

You can emulate the pagePiling.js behaviour by using the [parallax extension](https://alvarotrigo.com/fullPage/extensions/parallax.html) with the offset option set to 100% and placing your content in the `fp-bg` element so it moves at the speed of the background.

You can find a [demo here](https://alvarotrigo.com/fullPage/tumblr/) and the full article about the creating of that site [here](https://alvarotrigo.com/blog/how-create-tumblr-website-effect/).
